tests/test_e2e_31_of_lldp_loop_detection.py

```diff
@@ -0,0 +1,129 @@
+import requests
+from tests.helpers import NetworkTest
+import time
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api/kytos' % CONTROLLER
+
+
+class TestE2EOfLLDPLoopDetection:
+    net = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER, topo_name='looped')
+        cls.net.start()
+        cls.net.restart_kytos_clean()
+        #cls.net.wait_switches_connect()
+        #time.sleep(10)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def restart(self, _clean_config=False, _enable_all=False):
+
+        # Start the controller setting an environment in which the setting is
+        # preserved and avoid the default enabling of all elements
+        self.net.start_controller(clean_config=_clean_config, enable_all=_enable_all)
+        self.net.wait_switches_connect()
+
+        # Wait a few seconds 
+        time.sleep(10)
+
+    def test_001_loop_detection_disable_action(self):
+        """ This will test that given a looped topology, assuming that there is a loop
+        it's going to shutdown the interface. """
+
+        polling_time = 5
+
+        interface_id = "00:00:00:00:00:00:00:01:1"
+
+        # GET topology with the interface ensuring that it's enabled.
+        # Loop detection relies on the POLLING_TIME, by default is 3 seconds, 
+        # the tests will pretty much start with the interface down. 
+
+        #api_url = KYTOS_API + '/topology/v3/interfaces' 
```

Comment:
Could you remove this commented unused code?

Reply:
Ok